packages/react/src/Hero/Hero.tsx

```diff
@@ -105,9 +105,9 @@ type HeroDescriptionProps = {
 } & BaseProps<HTMLParagraphElement>
 
 const HeroDescription = forwardRef<HTMLParagraphElement, PropsWithChildren<HeroDescriptionProps>>(
-  ({size = '200', weight, children}: PropsWithChildren<HeroDescriptionProps>, ref) => {
+  ({size = '200', weight, children, ...rest}: PropsWithChildren<HeroDescriptionProps>, ref) => {
     return (
-      <Text ref={ref} className={styles['Hero-description']} as="p" size={size} weight={weight}>
+      <Text ref={ref} className={styles['Hero-description']} as="p" size={size} weight={weight} {...rest}>
```

Comment:
> One thing to consider is that when a custom class name is being used it will remove the Hero-description styles, this is existing behaviour on the HeroHeading therefore isn't a regression.

👋 @alexbuiltit - thanks for opening this PR, and drawing our attention to this potential regression ✨. 

To avoid losing the default styles, you could destructure the `className` as an argument before spreading the remaining properties in. The destuctured value can then be passed into a class name concat function (we typically use `clsx`).

Here's an example of how it's done [in another component](https://github.com/primer/brand/blob/main/packages/react/src/Button/Button.tsx#L160). Hope this helps!

Reply:
Hey @rezrah!

Okay great, didn't see you were already using `clsx`, I'll make that change today.

Reply:
Not sure how I missed the use of `clsx` given its already being imported in the `Hero` component 🤦

I've pushed the change now and update the changeset.